frontend/app-development/layout/SettingsModalButton/SettingsModal/components/Tabs/SetupTab/SetupTab.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { ReactNode } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useAppMetadataQuery } from 'app-development/hooks/queries';
+import { LoadingTabData } from '../../LoadingTabData';
+import { TabDataError } from '../../TabDataError';
+import { ErrorMessage } from '@digdir/design-system-react';
+import { TabHeader } from '../../TabHeader';
+import { SetupTabContent } from './SetupTabContent';
+
+export type SetupTabProps = {
+  org: string;
+  app: string;
+};
+
+/**
+ * @component
+ *    Displays the tab rendering the setup tab for an app
+ *
+ * @property {string}[org] - The org
+ * @property {string}[app] - The app
+ *
+ * @returns {ReactNode} - The rendered component
+ */
+export const SetupTab = ({ org, app }: SetupTabProps): ReactNode => {
+  const { t } = useTranslation();
+
+  const {
+    status: appMetadataStatus,
+    data: appMetadata,
+    error: appMetadataError,
+  } = useAppMetadataQuery(org, app);
+
+  const displayContent = () => {
+    switch (appMetadataStatus) {
+      case 'loading': {
```

Comment:
I think we do not need curly braces in cases :slightly_smiling_face:

Reply:
I'll fix that :)